docs/pages/blog/material-ui-v6-is-out.md

```diff
@@ -0,0 +1,391 @@
+This is a huge improvement over the previous version, which used padding and margin to create space.
+
+The responsive configuration has been consolidated into `size` and `offset` props for consistency.
+
+```jsx
+import Grid from '@mui/material/Grid2';
+
+<Grid container>
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+</Grid>;
+```
+
+### Package size reduction
+
+To align with React 19's removal of UMD builds, Material UI has also removed its UMD bundle.
+This results in a reduction of the `@mui/material` package size by 2.5MB, or 25% of the total package size.
+See [Package Phobia](https://packagephobia.com/result?p=@mui/material) for more details.
+Instead, we recommend using ESM-based CDNs such as [esm.sh](https://esm.sh/).
+For alternative installation methods, refer to the [CDN documentation](/material-ui/getting-started/installation/#cdn).
+
+## Experimental CSS extraction via Pigment CSS
+
+As you know, Material UI v5 uses Emotion as a default styling solution. As a runtime CSS-in-JS library, it has several trade-offs such as slower performance and larger bundle size.
+
+In v6, we introduce an opt-in integration with [Pigment CSS](https://github.com/mui/pigment-css), our new zero-runtime styling library, that will eliminate the runtime overhead while preserving similar APIs that you are already familiar with.
+
+:::warning
+Pigment CSS integration is still in the experimental stage. We recommend testing it out with our examples ([Next.js](https://github.com/mui/material-ui/tree/master/examples/material-ui-pigment-css-nextjs-ts) or [Vite](https://github.com/mui/material-ui/tree/master/examples/material-ui-pigment-css-vite)) and providing feedbacks to help us improve the feature.
+
+Then, if you're ready to integrate Pigment CSS into your project, follow the migration guide at the bottom of the page
+:::
+
+### React Server Components
+
+Once integrated with Pigment CSS, Material UI v6 provides a separate set of layout components, including `Grid`, `Container`, and `Stack`, that are compatible with React Server Components.
+
+```jsx
+import Grid from '@mui/material-pigment-css/Grid';
+import Container from '@mui/material-pigment-css/Container';
+import Stack from '@mui/material-pigment-css/Stack';
+```
+
+### Built-in sx prop support
+
+With Pigment CSS integration, all JSX elements support the `sx` prop out of the box.
+The `Box` component is no longer required to use the `sx` prop.
+
+```diff
+-import Box from '@mui/material/Box';
+
+-<Box component="img" sx={{ padding: 2 }} />
++<img sx={{ padding: 2 }} />
+```
+
+## React 19 support
+
+Material UI v6 is ready for React 19. We've been testing Material UI with the latest [React 19 RC](https://react.dev/blog/2024/04/25/react-19) versions to ensure compatibility once the stable release of React 19 is out.
+
+## Full-stack apps
```

Comment:
How do you feel about just "Toolpad Core" as a title? Would it spark more interest?

```suggestion
## Toolpad Core
```